language_detector/main.py

```diff
@@ -2,10 +2,31 @@
 
 """This is the entry point of the program."""
 
-from languages import LANGUAGES
+from .languages import LANGUAGES
 
 
 def detect_language(text, languages=LANGUAGES):
     """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    
+    ECounter = 0
+    SpanCounter = 0
+    GermCounter = 0
+    
+    text = text.lower()
+    new_list = text.split()
+    
+    
+    for word in new_list:
+      if word in LANGUAGES[0]["common_words"]:
+          SpanCounter += 1
+      elif word in LANGUAGES[1]["common_words"]:
+          GermCounter += 1  
+      elif word in LANGUAGES[2]["common_words"]:
+          ECounter += 1
+       
+    if max(SpanCounter,ECounter,GermCounter) == SpanCounter:
```

Comment:
Another note, if you use a dynamic counter, you can return the key with the max value in that dictionary instead of iterating through all the static counters. Dictionaries have a "get" function which was perfect for our solution.